src/middleware/validation.middleware.ts

```diff
@@ -5,29 +5,36 @@ import { getApplicationData, prepareData } from "../utils/application.data";
 import { NAVIGATION } from "../utils/navigation";
 import { DateOfBirthKey, StartDateKey, DateOfBirthKeys, StartDateKeys } from "../model/date.model";
 
+import { logger } from '../utils/logger';
+
 export function checkValidations(req: Request, res: Response, next: NextFunction) {
-  const errorList = validationResult(req);
-
-  if (!errorList.isEmpty()) {
-    const errors = formatValidationError(errorList.array());
-
-    // Bypass the direct use of variables with dashes that
-    // govukDateInput adds for day, month and year field
-    const dates = {
-      [DateOfBirthKey]: prepareData(req.body, DateOfBirthKeys),
-      [StartDateKey]: prepareData(req.body, StartDateKeys)
-    };
-
-    return res.render(NAVIGATION[req.path].currentPage, {
-      backLinkUrl: NAVIGATION[req.path].previousPage,
-      ...getApplicationData(req.session),
-      ...req.body,
-      ...dates,
-      errors
-    });
+  try {
+    const errorList = validationResult(req);
```

Comment:
Not related to these changes, but just wondering why there aren't any tests in the `validation.middleware.spec.ts` file related to the logic within this 'try catch' block. Is that logic tested somewhere else?

Reply:
👍 thanks. Guessed it hadn't been missed.